lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -49,8 +62,11 @@ def trace!(span, zipkin_env, &block)
       span.record(Trace::Annotation::SERVER_SEND)
     end
 
-    def trace_request_information(span, env)
-      span.record_tag(Trace::BinaryAnnotation::PATH, env['PATH_INFO'])
+    def trace_request_information(span, env, tags)
+      # tags is nil if we've been called by a zipkin-enabled service and the user hasn't
+      # specified tags to record on server receive.
+      return if tags.nil?
```

Comment:
updated following your recommendation but left the `return if tags.nil?` check so as not to needlessly create empty hash objects and call .each on them in what is going to be the 99% use case.